.travis/install.sh

```diff
@@ -3,4 +3,7 @@
 script_path=$(python -c "import os; import sys; print(os.path.realpath(sys.argv[1]))" "${BASH_SOURCE[0]}")
 source "${script_path%/*}/setup.sh"
 
-time ONNX_NAMESPACE=ONNX_NAMESPACE_FOO_BAR_FOR_CI pip install -v .
+export ONNX_BUILD_TESTS=1
+pip install protobuf numpy
+time ONNX_NAMESPACE=ONNX_NAMESPACE_FOO_BAR_FOR_CI python  setup.py bdist_wheel --universal --dist-dir .
+find . -maxdepth 1 -name "*.whl" -ls -exec pip install {} \;
```

Comment:
I guess you are doing this so the test binaries are generated in the source dir?
You can add `-b` to the `pip install` command in `script.sh` to point it to use a specified build path, so that later you can find and execute the test binaries.

Reply:
I ll re-test it later.. I tried set TMPDIR and --noclean somehow its not working.. I didnot remmeber if I used -b before

Reply:
Just tested the `-b` is not working.. only protobuf is retained in the given directory.. dont know the reason but log can be found https://api.travis-ci.org/v3/job/374997390/log.txt, command used is `pip install -b ${PWD}/.setuptools-cmake-build/ -v .`